tests/test_self_containment.cc.in

```diff
@@ -0,0 +1,27 @@
+/**
+ * \file   test_self_containment.cc.in
+ * \author \ref contributors
+ * \date   Created on August 21, 2019
```

Comment:
I don't care so much, but we should replace the date with the current one.

Reply:
Well, but that is the date when I created that file.